Carthage.md

```diff
@@ -0,0 +1,67 @@
+# Firebase Carthage
+
+## Context
+
+This page introduces and provides instructions for an **experimental** Firebase
+[Carthage](https://github.com/Carthage/Carthage) implementation. Based on
+feedback and usage, the Firebase team may decide to make the Carthage
+distribution official.
+
+## Carthage Installation
+
+[Homebrew](http://brew.sh/) is one way to install Carthage.
+
+```bash
+$ brew update
+$ brew install carthage
+```
+
+See the
+[Carthage page](https://github.com/Carthage/Carthage#installing-carthage) for
+more details and additional installation methods.
+
+## Carthage Usage
+
+- Create a Cartfile with a subset of the following components. Note that
+    **FirebaseAnalytics** must always be included.
+    - binary "https://dl.google.com/dl/firebase/ios/carthage/FirebaseABTestingBinary.json"
```

Comment:
Should these be wrapped in code blocks? Should this whole section be wrapped in a single codeblock (to emulate the Cartfile)?

Reply:
Yes. Done. Thanks!